website/docs/content/table-guidelines/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Table guidelines
+caption: Writing guidance for tabular data.
+description: Writing guidance for tabular data.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+related:
+  - components/table/table
+  - components/table/advanced-table
+---
+
+This documentation provides best practices for writing tabular content, such as with the [Table](/components/table/table) and [Advanced Table](/components/table/advanced-table) components.
+
+## General recommendations
+
+While we are not prescriptive about what goes into table cells, there are some best practices to consider:
+
+- We recommended keeping data within a column to one data type. Sorting on multiple data types is technically difficult and makes content reading confusing for assistive technologies.
+- While changing the text style/color within a cell is possible, we recommend only using Helios font styles and colors.
+
+## Icon usage
+
+Icons used within cells can help differentiate content, highlight additional metadata, increase the hierarchy of a value, or otherwise _enhance_ the text or value it is paired with. Use the outlined icon style by default and if contrast against other icons is important, use the filled style.
+
+!!! Dont
+
+Don’t use an icon as the sole communication method within a cell, even if the icon is explicit, e.g., a brand or service icon.
+
+![Icon within a table without a label](/assets/content/table-guidelines/icon-without-label.png)
+!!!
+
+!!! Dont
+
+We don’t recommend using an icon to indicate the status of an object, row, or resource. Instead, consider using a [Badge](/components/badge).
+
+![Icons being used for status](/assets/content/table-guidelines/icon-status.png)
+!!!
+
+### Service icons
+
+Use service icons within a cell to communicate the source or provider of a service.
+
+![Service icon within a table](/assets/content/table-guidelines/with-service-icon.png)
+
+### Grouping
+
+Use icons to communicate commonalities between values or that a value is part of a larger object or hierarchical structure.
+
+![Icons used for grouping in a table](/assets/content/table-guidelines/icon-grouping.png)
+
+### Product branding
+
+Use icons to communicate that a specific item is a HashiCorp product or resource.
+
+![Icon product branding](/assets/content/table-guidelines/icon-product-branding.png)
+
+!!! Dont
+
+Don’t use an [Icon Tile](/components/icon-tile) in place of an icon within a table cell.
+
+![Icon Tile within a table cell](/assets/content/table-guidelines/icon-tile-product-branding.png)
+!!!
+
+### Leading vs. trailing icons
+
+In general, we recommend using **leading icons** because the text following the icon will remain aligned and thus be easier for the user to scan.
```

Comment:
```suggestion
In general, we recommend using **leading icons** because the text following the icon will remain aligned and thus be easier for the user to visually scan.
```